consensus/polybft/contracts_initializer.go

```diff
@@ -12,12 +12,12 @@ import (
 	"github.com/umbracle/ethgo/abi"
 )
 
-const (
+var (
 	// Hydra modification: These are the minimal values that the ValidatorSet contract accept
```

Comment:
These are not the minimal values accepted anymore.
To make it clear, let's make the variable `initialMinStake` and let's add a constant minStakeLimit to be used everywhere in the tests, so we can know all operations work properly with the minimum min stake possible.